drainer/isyncer.go

```diff
@@ -0,0 +1,11 @@
+package drainer
```

Comment:
😕 what is the `i` in `isyncer.go` means?

Reply:
i means interface .... if the file name is confused you , we can change it .

Reply:
it seems ok